java/base-client/src/test/resources/token-response.json

```diff
@@ -1,4 +1,4 @@
 {
-  "expiresOn": 1645437609741,
-  "accessToken": "a-token"
-}
\ No newline at end of file
+  "expires_on": 1673262714,
```

Comment:
we actually both had the wrong format on these (should be expiry_on not expiryOn) AND it was epoch seconds and NOT MILLIseconds, which I assumed 🤔